src/components/browsePokemons/browsePokemons.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState, useEffect } from "react";
+import { Link, useNavigate, useLocation } from "react-router-dom";
+import Pagination from "@mui/material/Pagination";
+import {
+  Container,
+  Typography,
+  Box,
+  MenuItem,
+  FormControl,
+  Select,
+  Grid,
+  Card,
+  CardContent,
+  CardMedia,
+} from "@mui/material";
+import { usePokemonList } from "../../HooksEffect/hooksEffect";
+import LanguageSwitcher from "../languageSwitcher/languageSwitcher";
+import { useTranslation } from "react-i18next";
+import "./browsePokemons.css";
+
+function BrowsePokemons() {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  const getQueryParams = (query) => {
+    return new URLSearchParams(query);
+  };
+
+  const queryParams = getQueryParams(location.search);
+  const initialPage = parseInt(queryParams.get("page")) || 1;
+  const initialLimit = parseInt(queryParams.get("perPage")) || 10;
+
+  const [page, setPage] = useState(initialPage);
+  const [limit, setLimit] = useState(initialLimit);
+  const { pokemons, total, error } = usePokemonList(page, limit);
+
+  const handlePageChange = (event, value) => {
+    setPage(value);
+  };
+
+  const handleLimitChange = (event) => {
+    setLimit(Number(event.target.value));
+  };
+
+  useEffect(() => {
+    setPage(1);
+  }, [limit]);
+
+  useEffect(() => {
+    navigate(`?page=${page}&perPage=${limit}`);
+  }, [page, limit, navigate]);
+
+  if (error) {
+    return (
+      <Container maxWidth={false} className="container">
+        <Typography variant="h6" color="error">
+          {t("ErrorMessage")}
+        </Typography>
+      </Container>
+    );
+  }
+
+  return (
+    <Container maxWidth={false} className="container">
+      <LanguageSwitcher />
+      <Box
+        display="flex"
+        justifyContent="space-between"
+        alignItems="center"
+        mb={2}
+        className="header-box"
+      >
+        <Typography className="typography-h4">
+          {t("ChooseYourPokemon")}
+        </Typography>
+        <FormControl variant="outlined" className="form-control">
+          <Select
+            value={limit}
+            onChange={handleLimitChange}
+            displayEmpty
+            className="select"
+          >
+            <MenuItem value={5}>{t("itemPerPage", { count: 5 })}</MenuItem>

```

Comment:
please use map to difplay these Manu items, from the array
something like
menuItemOptions = [5,10,50]
and then map